src/test/config.sol

```diff
@@ -98,9 +98,9 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0xB945b792A1a87308C11573e7E9739f792dBCe226),        // populate with deployed spell if deployed
-            deployed_spell_created:         1661950301,        // use get-created-timestamp.sh if deployed
-            previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         0,        // use get-created-timestamp.sh if deployed
+            previous_spell:                 address(0xB945b792A1a87308C11573e7E9739f792dBCe226),        // supply if there is a need to test prior to its cast() function being called on-chain.
             office_hours_enabled:           true,              // true if officehours is expected to be enabled in the spell
```

Comment:
Office Hours looks Off on Exec Doc

Reply:
Fixed in [29c2995](https://github.com/makerdao/spells-mainnet/pull/270/commits/29c299577c1e3fca252f478cdf9f3f10e4dec27e)